src/lib/holocene/input/input.svelte

```diff
@@ -178,7 +178,7 @@
     </span>
     {#if maxLength && !disabled && !hideCount}
       <span
-        class="invisible text-right text-sm font-medium tracking-widest group-focus-within:visible"
+        class="invisible text-right text-xs tracking-widest group-focus-within:visible"
```

Comment:
> So it does at height to the input, wondering if we can make the count smaller text


@Alex-Tideman updated size for both `Input` and `Textarea` to be `text-xs` ⬇️ 

<img width="455" height="136" alt="Screenshot 2025-08-05 at 3 37 11 PM" src="https://github.com/user-attachments/assets/ef5f100f-8454-4801-b335-3c16b518d566" />
<img width="455" height="213" alt="Screenshot 2025-08-05 at 3 36 44 PM" src="https://github.com/user-attachments/assets/1867092e-3b19-4007-96ea-c01c4c95284d" />